### PR TITLE
Adding ide importing

### PR DESCRIPTION
**Pull request by vladbalanescu**
Contributes to #

Added `IdeImporting` from `ide-components`

#### Changelog

Use the following command to list changes or manually list out new, changed and
removed files.

```
M       packages/cdai/package.json
A       packages/cdai/src/component_helpers/FileHelper.js
A       packages/cdai/src/component_helpers/FileHelper.spec.js
A       packages/cdai/src/component_helpers/sleep.js
A       packages/cdai/src/components/IdeImporting/IdeImporting.js
A       packages/cdai/src/components/IdeImporting/IdeImporting.scss
A       packages/cdai/src/components/IdeImporting/IdeImporting.stories.js
A       packages/cdai/src/components/IdeImporting/IdeImporting.test.js
A       packages/cdai/src/components/IdeImporting/README.md
A       packages/cdai/src/components/IdeImporting/_storybook.scss
A       packages/cdai/src/components/IdeImporting/helpers.js
A       packages/cdai/src/components/IdeImporting/index.js
M       packages/cdai/src/index.js
M       packages/cdai/src/index.scss
```

**New**

`IdeImporting`

### CODE REVIEW

File: packages/cdai/src/index.scss [hunk 1]
```diff
@@ -9,5 +9,6 @@
 
 @import 'components/IdeButton/IdeButton';
 @import 'components/IdeEmptyState/IdeEmptyState';
+@import 'components/IdeEmptyState/IdeImporting';
```
Comment (asfordmatt) [inline packages/cdai/src/index.scss:12]: shouldn't that be `'components/IdeImporting/IdeImporting'` ?

**Comment by davidmenendez** [comment]
Hey @vladbalanescu is this for #30 ? If so I was assigned this issue and already have a PR open for this https://github.com/carbon-design-system/ibm-cloud-cognitive/pull/163

**Comment by asfordmatt** [comment]
Hi @davidmenendez this PR is just moving across the existing CDAI Import component as-is.

**Comment by davidmenendez** [comment]
I guess i'm confused about what the difference is between the two components. is the ide import supposed to be separate from the component in the issue that i linked? either way they are both essentially the same thing, so why wouldn't we have a single reusable component?

**Comment by asfordmatt** [comment]
Many of the designs have changed significantly, but this component does look rather similar. The issue does refer to the existing code:
https://github.com/carbon-design-system/ibm-cloud-cognitive/issues/30#issuecomment-669259813

**Comment by davidmenendez** [comment]
right, but @lee-chase assigned me #30 and i've been working on that component for some time. perhaps we need to get on slack to discuss this.

File: packages/cdai/src/index.scss (hunk 1 above)
Comment (vladbalanescu) [reply packages/cdai/src/index.scss:12]: yep, you're right 👍

Comment (vladbalanescu) [reply packages/cdai/src/index.scss:12]:
```suggestion
@import 'components/IdeImporting/IdeImporting';
```

**Comment by vladbalanescu** [comment]
There are 2 different components @davidmenendez. The entire integration portfolio of IBM is using components from: https://pages.github.ibm.com/cloud-integration-design/ide-components/?path=/story/ide-home-experience--specifying-custom-card-content, hence we're tackling this task in 2 stages:
- move accross the components as are from `ide-components` into this new repo
- converge them all into this new repo into a single set in order to avoid duplication

Base line: yep, they might be similar, but for a flawless and seemless experience, we need integration products to not be affected by this change in any way, hence we're moving them accross as an exact copy first, then we can discuss merging these into a single set of components. 

I hope this makes sense. Feel free to get in touch with me or @asfordmatt directly if you do require any more clarification on this PR.

**Review by asfordmatt — APPROVED** [review]
LGTM

**Comment by davidmenendez** [comment]
Ok. I was unaware of this. Thank you for sharing that background information.

**Pull request merged**